Makefile

```diff
@@ -253,11 +253,11 @@ install-operator: install-apis
 install-strategies: install-apis
 	@hack/shipwright-build.sh install strategies
 
-local: install-strategies build
+local: vendor install-strategies
```

Comment:
The `build` target does a `vendor`.  So I decided to explicitly keep the `vendor`, because I want to make sure that the Go modules `vendor` directory is in sync with the Go modules files before the operator SDK does the `go build`.

Reply:
I see, u re right!